justega247/airtech
# #168165079: Create the endpoint for flight booking

#### What does this PR do?
- This PR is for the creation of the endpoint for users to be able to book flights
#### Description of Task to be completed?
- write code to implement the endpoint for booking a flight
- make modifications to the model to alter the booking functionality
- write tests to confirm the functionality works as expected
#### How should this be manually tested?
- clone this branch using **git clone -b ft-book-flight-endpoint-168165079 
  --single-branch https://github.com/justega247/airtech.git**
- Navigate to the root directory and run **python manage.py runserver ** to start up the application
- Using an app like **postman** or any other related means for accessing exposed endpoints, make a **POST** request to **http://127.0.0.1:8000/booking/** with valid details to create a booking
- You should get a successful creation if valid details have been provided
#### What are the relevant pivotal tracker stories?
[#168165079](https://www.pivotaltracker.com/story/show/168165079)
